src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -22,9 +35,81 @@ export class LwcGenerationCommand {
                             panel.dispose();
                             return resolve();
                         }
+                    },
+                    {
+                        type: 'getQuickActionStatus',
+                        action: async (_panel, data, callback) => {
+                            // TODO: Hook this up to function that parses landing_page.json.
+                            const sobjects = [
+                                'Account',
+                                'Contact',
+                                'Opportunity',
+                                'SomeOther'
+                            ];
+                            if (callback) {
+                                const quickActionStatus =
+                                    await LwcGenerationCommand.checkForExistingQuickActions(
+                                        sobjects
+                                    );
+                                callback(quickActionStatus);
+                            }
+                        }
                     }
                 ]
             );
         });
     }
+
+    static async checkForExistingQuickActions(
+        sobjects: string[]
+    ): Promise<SObjectQuickActionStatus> {
+        return new Promise<SObjectQuickActionStatus>(async (resolve) => {
+            const results: SObjectQuickActionStatus = { sobjects: {} };
+            results.sobjects = {};
```

Comment:
You don't need this, right? looks like it's initialized as such in your declaration.